lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -279,20 +280,20 @@ def main():
     cert_chain = module.params.get('cert_chain')
     dup_ok = module.params.get('dup_ok')
     if state == 'present':
-        try:
+        if os.path.exists(module.params.get('cert')):
```

Comment:
My understanding is exists will return true for a folder, isfile will not.  So probably yes. isfile will be the best fit.